identity/client/src/pages/users/List.tsx

```diff
@@ -66,7 +71,7 @@ const List: FC = () => {
     <Page>
       <EntityList
         title={t("Users")}
-        data={users}
+        data={userSearchResults?.items || []}
```

Comment:
I think here we can assume that `userSearchResults?.items` is available as we have checked the length in previous if at line 45.

Reply:
I thought this too but when I leave out the `|| []` it complains that `User[] | undefined` is not assignable to `User[] | null`, I'll see if its me doing something wrong here

Reply:
I see, Can you try with `userSearchResults!.items! `or sth like this?

Reply:
That done the trick :)